CHANGELOG.md

```diff
@@ -10,7 +10,9 @@
 
 ##### Bug Fixes
 
-* None.  
+* Ignore path when checking vendored_libraries `lib[name].a` format.
```

Comment:
This is a bit misleading, maybe change to `support **/lib*.a in vendored_libraries when linting`

Reply:
Ah I see what you mean by ignoring the path